### DISCUSSION TITLE
GitHub Copilot atop VSCode won't connect to account over SSH/Tunnel workspace, but does so in local workspace...

### DISCUSSION THREAD

**Discussion by nmvega** [microsoft/vscode-discussions#2449] [Extension Development QnA] [answered]
Hello Friends:

I have an activated free **GitHub Copilot** account.

The below screenshot shows **VSCode** in an **SSH/Tunnel workspace context**, and you'll notice that the **right-hand panel** shows the **Use Copilot For Free** button. Clicking that button briefly indicates "**Getting copilot ready...**" in the **status bar**, but it doesn't completing the action, leaving right-hand panel unchanged. No error is emitted.

Conversely, **VSCode** in the **local workspace context** doesn't have this problem. Both the r**ight-hand panel** as well as the **Chat CLI** below the **Linux Bash** prompt both work.

Any ideas why this request doesn't complete in the **SSH/Tunnel workspace context**? Thank you in advance!

![Screenshot from 2024-12-18 18-09-46](https://github.com/user-attachments/assets/d370d1d3-8070-48d4-b243-87eb5f0a0a87)

**Comment by maremango** [comment, marked as answer]
https://github.com/microsoft/vscode/issues/236537

**Reply by nmvega** [reply]
@maremango Thank you very much! ☺️
